enclave-runtime/src/initialization.rs

```diff
@@ -78,7 +81,10 @@ use its_sidechain::{
 use log::*;
 use primitive_types::H256;
 use sp_core::crypto::Pair;
-use std::{string::String, sync::Arc};
+use std::{fs, string::String, sync::Arc};
+
+const ENCLAVE_CERTIFICATE_FILE_PATH: &str = "cert.pem";
+const ENCLAVE_CERTIFICATE_PRIVATE_KEY_PATH: &str = "private_key.pem";
```

Comment:
We have a separate crate `itp-settings`, that contains constants like these. Move them there.